src/main/java/core/basesyntax/services/operations/Supply.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.services.operations;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Supply implements Operable {
```

Comment:
```suggestion
public class SupplyOperation implements Operable {
```
let's make naming cosistent